src/tt21100.toit

```diff
@@ -64,33 +63,33 @@ class Action:
     else:
       return "Action is not supported"
 
-class Driver:
-  ADDRESS_/int ::= 0x24
+class Tt21100:
+  static I2C_ADDRESS ::= 0x24
 
   features/int
   width/int
   height/int
   ready_pin/gpio.Pin
   device/i2c.Device
 
-  constructor .width/int .height/int i2c_bus/i2c.Bus .ready_pin/gpio.Pin --frequency/int=400_000 --flags/int=0:
+  constructor i2c_bus/i2c.Bus .ready_pin/gpio.Pin --frequency/int=400_000 --flags/int=0 --.width/int --.height/int:
```

Comment:
Forgot this in my first review.
I think that we usually only take the device and not the bus as argument for drivers.
Not a big change, but would make it more consistent.

Reply:
Also makes it possible to use the same driver for alternative I2C addresses.